### ISSUE TITLE
Max line-width violated following refactor.

### ISSUE THREAD

**Issue by somada141** [sourcery-ai/sourcery#11]
### Issue description or question

Refactoring doesn't respect the line-widths.

Here's an example. Before:

```
                val = []
                for sub_attr_value in attr_value:
                    val.append(sub_attr_value.to_dict(
                        deep=deep,
                        serialisable=serialisable
                    ))
```

After:

```
val = [sub_attr_value.to_dict(deep=deep, serialisable=serialisable) for sub_attr_value in attr_value]
```

This isn't a major deal but will then require re-refactoring or running `black` to bring the code back in shape. 

### Code editor or IDE name and version

Pycharm v2019.2.1

### OS name and version

OSX 10.14.6

**Comment by Hellebore** [comment]
Thanks for raising - this is a tricky area for us to get right so we will have a think.

**Comment by somada141** [comment]
You may be able to pull off some trick where you isolate the snippet, write it in a temporary file, and invoke `black` to auto-format within a given line-width but it sounds hacky :P

**Comment by Hellebore** [comment]
Finally got this fixed! Now live in plugin version 0.3.1
